src/main/xar-resources/data/lucene/lucene.xml

```diff
@@ -318,27 +318,33 @@
                     These parameters must match a <code>Constructor</code> signature on the
                     underlying Java class of the Analyzer, please review the Javadoc for the
                     Analyzer that you wish to configure.</para>
-                <para>We currently support passing the following types: </para>
+                <para>We currently support passing the following types:</para>
                 <itemizedlist>
                     <listitem>
-                        <para> <code>String</code> (default if no type is specified)</para>
+                        <para><code>String</code> (default if no type is specified)</para>
                     </listitem>
                     <listitem>
-                        <para> <code>java.io.FileReader</code> (since Lucene 4) or <code>file</code>
+                        <para><code>String[]</code> (since eXist-db 5.4.0)</para>
+                    </listitem>
+                    <listitem>
+                        <para><code>char[]</code> (since eXist-db 5.4.0)</para>
+                    </listitem>
+                    <listitem>
+                        <para><code>java.io.FileReader</code> (since Lucene 4) or <code>file</code>
```

Comment:
not sure what the reader is to make of the metion of `lucena 4` here, I'd argue that changing exist-db's lucena version is way beyond target audience skill level

Reply:
I didn't add that, it was already there in the documentation ;-)
Are you suggesting it should be removed?

Reply:
yes it adds nothing substantial, and only serves to confuse. Unless you happen to know since what exist version the `file` is supported, which would provide actionable information, just remove that mention

Reply:
@duncdrum Done!

Reply:
great thanks please see the rest of my comments